test/e2e/test_nginx_entrypoint.py

```diff
@@ -22,6 +23,10 @@ def test_nginx_sigterm_handling(host, container):
 @pytest.mark.nginx_e2e
 @pytest.mark.parametrize('container', [{'env': {'NGINX_PORT': '5556'}, 'port': '5556'}], indirect=True)
 def test_nginx_can_host_different_ports(host, container):
+    start = time.time()
+    while host.run('docker exec -t {} sh -c "wget http://127.0.0.1:5556/"'.format(container)).rc is not 1 and time.time() - start < 13:
+        time.sleep(1)
```

Comment:
Why is this suddenly necessary, and why wait for 13 seconds specifically?

Reply:
It's a timing issue, most of the time a one second sleep works, sometimes it needs more. 13 is just a random number